android/app/src/main/java/org/uwaterloo/subletr/enums/LocationRange.kt

```diff
@@ -0,0 +1,16 @@
+package org.uwaterloo.subletr.enums
+
+import androidx.annotation.StringRes
+import org.uwaterloo.subletr.R
+
+enum class LocationRange(@StringRes val stringId: Int) {
```

Comment:
I think this is fine for now, but we are going to eventually get this from backend I presume?
@andr-dev